dubbo-filter/dubbo-filter-validation/src/main/java/org/apache/dubbo/validation/support/jvalidation/JValidation.java

```diff
@@ -28,12 +31,24 @@
 public class JValidation extends AbstractValidation {
 
     /**
-     * Return new instance of {@link JValidator}
+     * Return new instance of {@link Validator}
      * @param url Valid URL instance
-     * @return Instance of JValidator
+     * @return Instance of Validator
      */
     @Override
     protected Validator createValidator(URL url) {
-        return new JValidator(url);
+        List<Class<? extends Validator>> validatorList = Arrays.asList(JValidator.class, JValidatorNew.class);
```

Comment:
Use `Class.forName`

Reply:
Did you meaning loading `JValidator.class` and `JValidatorNew.class` classes with `Class.forName(String)` like this? Or could you please provided more detail info

```
try {
    Class<? extends Validator> validCls = (Class<? extends Validator>) Class.forName("org.apache.dubbo.validation.support.jvalidation.JValidator");
    Class<? extends Validator> newValidCls = (Class<? extends Validator>) Class.forName("org.apache.dubbo.validation.support.jvalidation.JValidatorNew");
} catch (Throwable ignore) {
}
```

By the way, the `JValidator.java#316` and `JValidatorNew.java#316` need to be modified together?

Reply:
Yes.
A more conscise way is to use `org.apache.dubbo.common.extension.Activate#onClass` and create two filter for `javax` and `jakarta`